labels/de-DE.json

```diff
@@ -527,6 +527,8 @@
   "quoteExpiredBuyerBlob2": "Bitte senden Sie eine neue Anfrage oder wenden Sie sich an Ihren Vertriebsmitarbeiter, um Hilfe zu erhalten",
   "quoteExpiredSeller": "Bonjour, <br /> <br /> <br /> le devis \"{0}\" (# {1}) a expiré le",
   "quoteHistory": "Zitat Geschichte",
+  "quoteShippingMethods": "Versandarten",
```

Comment:
Deepak, read my previous comment, where I mentioned these keys should be in alphabetical order. All the keys in this file are in alphabetical order. see "quoteExpiredBuyerBlob2" is first and then "quoteExpiredSeller".

Reply:
done